### PR TITLE
Updated Ollama cost models to include LLaMa3 and Mistral/Mixtral Instruct series

### PR DESCRIPTION
**Pull request by kmheckel**
## Title

Updated Ollama cost models to include LLaMa3 and Mistral Instruct series

## Relevant issues

None currently open.

## Type

🆕 New Feature

## Changes

1. Updated model_prices_and_context_window.json to include chat endpoints for LLaMa3 and Mistral/Mixtral Intstruct.
This enables users to use any LLaMa3/Mistral/Mixtral finetunes in chat mode by doing `ollama cp <finetune name> <llama3/mistral-7B-Instruct-v0.1>` for example. 

2. Small error message modifcation to include model and provider name in the error message for custom LLM providers. This makes debugging easier for users as they can see if it's just a typo or if they need to go update the cost model file.

## Testing

To test, instructions from this AutoGen tutorial mostly suffice: https://microsoft.github.io/autogen/docs/topics/non-openai-models/local-litellm-ollama/

1. Install Ollama
2. Pull LLama3
3. `ollama serve`
4. litellm --model ollama_chat/llama3
5. In python, create a LiteLLM completion object and interact with it.

## Notes

From my understanding, this pull request removes the necessity to specify custom pricing models from command line via a config.yaml: https://docs.litellm.ai/docs/proxy/custom_pricing

Tested locally to confirm functionality, with streaming set to False as part of the LiteLLM interface to DataDreamer.
https://github.com/datadreamer-dev/DataDreamer/blob/0.35.0/src/llms/_litellm.py

No concerns or substantial modifications; update simply adjusts metadata for newer Ollama models.

## Pre-Submission Checklist (optional but appreciated):

- [x] I have included relevant documentation updates (stored in /docs/my-website)

## OS Tests (optional but appreciated):
Not tested on other operating systems but this change won't break/doesn't fix other open issues with Ollama-->LiteLLM.

- [ ] Tested on Windows
- [ ] Tested on MacOS
- [x] Tested on Linux

### CODE REVIEW

**Comment by kmheckel** [comment]
@ishaan-jaff If you wouldn't mind review this sometime soon it would be helpful so that I don't have to keep working of a local fork of LiteLLM to use Ollama. Thanks!

File: model_prices_and_context_window.json [hunk 1]
```diff
@@ -2749,6 +2749,24 @@
         "litellm_provider": "ollama",
         "mode": "completion"
     },
+    "ollama/llama3": {
+        "max_tokens": 8192,
+        "max_input_tokens": 8192,
+        "max_output_tokens": 8192,
+        "input_cost_per_token": 0.00000010,
+        "output_cost_per_token": 0.00000010,
```
Comment (ghost) [inline model_prices_and_context_window.json]: does ollama have a hosted endpoint ? why is there a cost here

Comment (kmheckel) [reply model_prices_and_context_window.json]: Good catch, an oversight during copy&paste - I'll fix that.

File: litellm/utils.py [hunk 2]
```diff
@@ -6620,7 +6620,7 @@ def _get_max_position_embeddings(model_name):
             raise Exception()
     except:
         raise Exception(
-            "This model isn't mapped yet. Add it here - https://github.com/BerriAI/litellm/blob/main/model_prices_and_context_window.json"
+            f"Model {model} from provider {custom_llm_provider} isn't mapped yet. Add it here - https://github.com/BerriAI/litellm/blob/main/model_prices_and_context_window.json"
```
Comment (ghost) [inline litellm/utils.py]: @kmheckel this would cause another error - if custom_llm_provider isn't found from 'get_llm_provider'

Comment (kmheckel) [reply litellm/utils.py]: I'll remove the custom_llm_provider reference in that case.

**Review by ishaan-jaff — APPROVED** [review]

**Pull request merged**